sova/core/steps/spec.py

```
@@ -361,12 +361,11 @@ async def execute(self, ctx: ExecutionContext) -> StepResult:
         has_questions = _text_has_open_questions(text)
         spec_complexity = _extract_complexity(text)
 
-        # Auto-approve specs at or below threshold when they have no open questions.
+        # Auto-approve simple specs without open questions.
         # Auto-approved specs chain directly to developer via handoff with auto_execute=True,
         # exiting the researcher pipeline early (skip-to-role pattern).
-        threshold = ctx.config.spec.auto_approve_threshold
-        if threshold != "none" and not has_questions:
-            if _complexity_rank(spec_complexity) <= _complexity_rank(threshold):
+        if ctx.config.spec.auto_approve_simple and not has_questions:
+            if _complexity_rank(spec_complexity) <= _complexity_rank("simple"):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)